hokusai/commands/development.py

```diff
@@ -1,14 +1,15 @@
 import os
 import signal
 
+from hokusai import CWD
 from hokusai.lib.command import command
 from hokusai.lib.config import config
 from hokusai.lib.common import print_green, shout, EXIT_SIGNALS
 from hokusai.lib.exceptions import HokusaiError
 
 @command()
 def dev_start(build, detach):
-  docker_compose_yml = os.path.join(os.getcwd(), 'hokusai/development.yml')
+  docker_compose_yml = os.path.join(CWD, 'hokusai/development.yml')
```

Comment:
should these `hokusai/` change to `HOKUSAI_CONFIG_DIR`?

Reply:
Good call!